cdap-docs/reference-manual/source/http-restful-api-v3/introduction.rst

```diff
@@ -0,0 +1,179 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: HTTP RESTful Interface to the Cask Data Application Platform
+    :copyright: Copyright © 2014 Cask Data, Inc.
+
+.. _http-restful-api-introduction:
+
+===========================================================
+Introduction
+===========================================================
+
+.. highlight:: console
+
+.. _http-restful-api-conventions:
+
+Conventions
+============
+
+In this API, *client* refers to an external application that is calling CDAP using the HTTP interface.
+*Application* refers to a user Application that has been deployed into CDAP.
+
+.. rubric:: Base URL
+
+All URLs referenced in this API (with the exception of those in the 
+Namespace API) have this base URL::
+
+  http://<host>:<port>/v3/namespaces/<namespace-id>
```

Comment:
I think that will increase confusion. I think that we should find some way that any code that worked under v2 should, if you replace ``/v2/`` with ``/v3/namespaces/default/``, it should still work. That should be possible, no? The router would know if an API was not namespaced, and could it just re-route it appropriately? Or would there be problem with people expecting something to be namespaced that actually wasn't?

Another approach would be that certain APIs would use ``system`` as the namespace rather than ``default``; however, that would require users to know which APIs use that and which don't...

I am thinking we need to add a "Summary" page that combines all RESTful APIs into one, so that people can scan the list for them. I don't like duplicating documentation, but I think this would be helpful as a quick reference guide.
Thoughts?